usage/rest-api/src/main/java/brooklyn/rest/api/ServerApi.java

```diff
@@ -87,6 +88,11 @@ public void shutdown(
             multiValueResponse = false)
     public String getStatus();
 
+    @GET
+    @Path("/getConfig/{configKey}")
+    @ApiOperation(value = "Get the value of the specified config key from brooklyn properties")
+    public String getConfig(@PathParam("configKey") final String configKey);
```

Comment:
Also say that it will give 404 if not found (again, see `EntityConfigApi.get` for example).